src/components/Common/Button.tsx

```diff
@@ -1,24 +1,35 @@
 import { ReactNode } from "react";
 import cx from "classnames";
-import "./Button.css";
+import "./Button.scss";
+
+export enum ButtonType {
+  Transparent,
+}
```

Comment:
i think it's okay to have `type ButtonType = "transparent"` so we can use it as <Button type="transparent">...</Button>, importing enum from other places may be bit overhead

Reply:
Sure!